docs/operation/cluster_and_service_configuration.rst

```diff
@@ -147,6 +150,61 @@ To add a new static user, perform the following operations:
 #. Verify that the user has been successfully added and you can log in to the
    MetalK8s UI using the new email and password.
 
+.. _Change-dex-user-password:
+
+Change Password for local static user
+"""""""""""""""""""""""""""""""""""""
+
+To change the password of an existing user, perform the following operations:
+
+#. Generate a bcrypt hash of the new password.
+
+   - Startup the MetalK8s utility container by refering to
+     :ref:`this procedure <Troubleshooting-container>`
+
+   - To generate the bcrypt hash, run the following command while replacing
+     the <password> field with your plaintext password.
+
+   .. code-block:: shell
+
+      root@bootstrap $ crictl exec -it \
```

Comment:
See above. Also, de-duplicate this section with the one above. The way to calculate a password hash is always the same.

Reply:
Done!